packages/components/src/menu/stories/index.story.tsx

```diff
@@ -22,7 +22,7 @@ import { createSlotFill, Provider as SlotFillProvider } from '../../slot-fill';
 import { ContextSystemProvider } from '../../context';
 
 const meta: Meta< typeof Menu > = {
-	title: 'Components (Experimental)/Menu',
+	title: 'Components (Experimental)/Actions/Menu',
```

Comment:
This will, unfortunately, invalidate the previous storybook URL of this story.

We might want to use `id: 'components-experimental-menu'` or `id: 'components-menu'`. It's a private component, and we recently renamed it from `DropdownMenuV2`, so that should be fine.